CONTRIBUTING.md

```diff
@@ -9,3 +9,21 @@ The develop of the Ottawa Data Model mostly occurs with informal working groups
 Working groups propose changes through a pull request to `dev`. Initial review for the PR is from previous contributors. Following there is a pull request from `dev` to `main` for proposed changes to the next version of the Ottawa Data Model. 
 
 We aim for a one-week community review period from `dev` to `main` during the initial agile development period. We welcome community review from anyone in the wastewater surveillance community. The ODM must work for your needed to be successful. Please `watch` this GH repo to ensure you are aware of proposed changed. 
+
+## Repo Organization
+
+### Naming Conventions
+
+* Use [kebab-case](https://www.theserverside.com/definition/Kebab-case#:~:text=Kebab%20case%20%2D%2D%20or%20kebab,properly%20convey%20a%20resource's%20meaning.) when naming files and folder making sure to use lower case
```

Comment:
I agree with @DougManuel that this would require us to go through and rename a few objects. Not a dealbreaker though, and probably something that needs to be done sooner rather than later.